src/main/java/com/twcable/grabbit/tools/jobstarter/HostAndJobIds.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2016 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twcable.grabbit.tools.jobstarter;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.reactivestreams.Publisher;
+
+import java.net.URI;
+
+/**
+ * Tuple of a host URI and a Publisher of job ids.
+ */
+@SuppressWarnings({"WeakerAccess", "checkstyle:VisibilityModifier", "checkstyle:NeedBraces"})
+public class HostAndJobIds {
+    public final URI uri;
```

Comment:
Just kidding. :-)  By bringing in [Lombok](https://projectlombok.org/features/) I was able to make all that boilerplate go away.